README.md

```diff
@@ -1,15 +1,27 @@
-# Interactive Message Adapter for Node and Express
+# Slack Interactive Messages for Node
 
 [![Build Status](https://travis-ci.org/slackapi/node-slack-interactive-messages.svg?branch=master)](https://travis-ci.org/slackapi/node-slack-interactive-messages)
 [![codecov](https://codecov.io/gh/slackapi/node-slack-interactive-messages/branch/master/graph/badge.svg)](https://codecov.io/gh/slackapi/node-slack-interactive-messages)
 
-The adapter provides a simplified API to route message actions and options in your app. It handles
-common tasks and best practices so that you don't need to.
+Build your Slack Apps with rich and engaging user interactions using buttons, menus, and dialogs.
+The packages makes it easy to start with sensible and secure defaults.
```

Comment:
Small revision: "\nThis package will help you start with sensible and secure defaults."